#diff --git a/misc/skins/Makefile.am b/misc/skins/Makefile.am
#index feebe48ef..d2f066e06 100644
#--- a/misc/skins/Makefile.am
#+++ b/misc/skins/Makefile.am
#@@ -12,6 +12,7 @@ skin_DATA = \
# 	gray-orange-blue256.ini \
# 	mc46.ini \
# 	modarcon16-defbg.ini \
#+	modarcon16-defbg2.ini \
# 	modarcon16.ini \
# 	modarcon16root-defbg.ini \
# 	modarcon16root.ini \
diff --git a/misc/skins/Makefile.am b/misc/skins/Makefile.am
index 74d8f82d9..99435fb49 100644
--- a/misc/skins/Makefile.am
+++ b/misc/skins/Makefile.am
@@ -13,6 +13,7 @@ skin_DATA = \
 	julia256.ini \
 	mc46.ini \
 	modarcon16-defbg.ini \
+	modarcon16-defbg2.ini \
 	modarcon16.ini \
 	modarcon16root-defbg.ini \
 	modarcon16root.ini \
--- /dev/null	2020-01-18 17:25:50.988930994 +0100
+++ new/misc/skins/modarcon16-defbg2.ini	2019-05-06 01:07:16.145318511 +0200
@@ -0,0 +1,189 @@
+#
+# modarin256: a moderate-dark-independent 256-color skin for MidnightCommander.
+#
+# This skin is using colors independent from the default 16 terminal colors,
+# i.e., no matter how the 16 standard colors are configured, the appearance
+# of Midnight Commander will not be affected.
+#
+# There are 4 variants of this skin:
+#
+# modarin256
+#   - default skin, no standard colors used.
+#
+# modarin256-defbg
+#   - like modarin256, but using the default background color for core, editor,
+#     viewer, menu and popupmenu backgrounds, for use with (semi-)transparent
+#     background colors.
+#
+# modarin256root
+# modarin256root-defbg
+#   - like the other two skins, but for the root user, with red colors used
+#     instead of green colors for things like select/focus/statusbar etc.,
+#     to keep reminding the user that mc is running as root.
+#
+# Fallback skins for 16-color terminals (poor man's skins):
+#
+# modarcon16
+# modarcon16-defbg
+# modarcon16root
+# modarcon16root-defbg
+#   - like everything running in a 16-color terminal, these skins look ugly
+#     and are no subsitute for the 256-color versions. As some terminals don't
+#     support using dark gray as background color, i used a black background
+#     and adjusted the remaining colors accordingly.
+#
+# 256-color skins require mc version 4.8 or newer.
+#
+# Author: Oliver Lange <modarin@bloody.in-berlin.de>
+#
+# Version: 1.2
+#
+
+[skin]
+    description = MoDarCon16-DefaultBG (user)
+    256colors = false
+
+[Lines]
+    horiz = ─
+    vert = │
+    lefttop = ┌
+    righttop = ┐
+    leftbottom = └
+    rightbottom = ┘
+    topmiddle = ┬
+    bottommiddle = ┴
+    leftmiddle = ├
+    rightmiddle = ┤
+    cross = ┼
+    dhoriz = ═
+    dvert = ║
+    dlefttop = ╔
+    drighttop = ╗
+    dleftbottom = ╚
+    drightbottom = ╝
+    dtopmiddle = ╤
+    dbottommiddle = ╧
+    dleftmiddle = ╟
+    drightmiddle = ╢
+
+[core]
+    _default_ = color7;default
+    selected = color15;color6
+    marked = color11;;bold
+    markselect = color11;color6;bold
+    gauge = color15;color6
+    input = color15;color0
+    inputmark = color11;color6;bold
+    inputunchanged = ;color0
+    commandlinemark = color11;color6;bold
+    reverse = color0;color6
+# ^ current dir at top of panel
+    header = color3
+    disabled = color8;color7
+    #inputhistory =
+    #commandhistory =
+
+[dialog]
+    _default_ = color0;color7
+    dhotnormal = color3
+    dfocus = color11;color6;bold
+    dhotfocus = color3;color6
+    dtitle = color11;;bold
+
+[error]
+    _default_ = color15;color1
+    errdfocus = color11;color6;bold
+    errdhotnormal = color13
+    errdhotfocus = color13;color6
+    errdtitle = color11;;bold
+
+[filehighlight]
+    directory = color15;;bold
+    executable = color10
+    symlink = color12
+    hardlink =
+    stalelink = color9
+    device = color13
+    special = color3
+    core = color1
+    temp = color8
+    archive = color3
+    doc = color14
+    source = color6
+    media = color5
+    graph = color13
+    database = color6
+
+[menu]
+    _default_ = color7;default
+    menusel = color11;color6;bold
+    menuhot = color3
+    menuhotsel = color3;color6
+    menuinactive = color8
+
+[popupmenu]
+    _default_ = color7;default
+    menusel = color11;color6;bold
+    menutitle = color11;;bold
+
+[buttonbar]
+    button = color7
+    hotkey = color15;color3
+
+[statusbar]
+    _default_ = color15;color6
+
+[help]
+    _default_ = color0;color7
+    helpitalic = color10;;bold
+    helpbold = color5
+    helplink = color14
+    helpslink = color11;color6;bold
+    helptitle = color11;;bold
+
+[editor]
+    _default_ = color7;default
+    editbold = color15;;bold
+    editmarked = color11;color6;bold
+    editwhitespace = color12;color4
+    editlinestate = color6;color0
+    bookmark = color0;color7
+    bookmarkfound = color0;color7
+    editrightmargin = color0;color7
+
+[viewer]
+    _default_ = color7;default
+    viewbold = ;;bold
+    viewunderline = ;;underline
+    viewselected = color11;color6;bold
+
+[diffviewer]
+    changedline = color15;color4
+    changednew = color15;color6
+    changed = color15;color6
+    added = color0;color7
+    removed = color8;color0
+    error = color15;color1
+
+[widget-common]
+    sort-sign-up = ↑
+    sort-sign-down = ↓
+
+[widget-panel]
+    hiddenfiles-sign-show = •
+    hiddenfiles-sign-hide = ○
+    history-prev-item-sign = «
+    history-next-item-sign = »
+    history-show-list-sign = ^
+
+[widget-scollbar]
+    first-vert-char = ↑
+    last-vert-char = ↓
+    first-horiz-char = «
+    last-horiz-char = »
+    current-char = ■
+    background-char = ▒
+
+[widget-editor]
+    window-state-char = *
+    window-close-char = X
